finality-aleph/src/network/manager/service.rs

```diff
@@ -419,21 +435,30 @@ impl<D: Data> IO<D> {
     ) -> Result<(), Error> {
         let mut maintenance = interval(Duration::from_secs(MAINTENANCE_PERIOD_SECONDS));
         loop {
+            trace!(target: "aleph-network", "Manager Loop started a next iteration");
             tokio::select! {
-                maybe_command = self.commands_from_user.next() => match maybe_command {
+                maybe_command = self.commands_from_user.next() => {
+                    trace!(target: "aleph-network", "Manager Loop command case");
+                    match maybe_command {
                     Some(command) => match service.on_command(command).await {
                         Ok(to_send) => self.send(to_send)?,
                         Err(e) => warn!(target: "aleph-network", "Failed to update handler: {:?}", e),
                     },
                     None => return Err(Error::CommandsChannel),
+                }
                 },
-                maybe_message = self.messages_from_user.next() => match maybe_message {
+                maybe_message = self.messages_from_user.next() => {
+                    trace!(target: "aleph-network", "Manager Loop message from user case");
+                    match maybe_message {
                     Some((message, session_id, recipient)) => for message in service.on_user_message(message, session_id, recipient) {
                          self.send_data(message)?;
                     },
                     None => return Err(Error::MessageChannel),
+                }
                 },
-                maybe_message = self.messages_from_network.next() => match maybe_message {
+                maybe_message = self.messages_from_network.next() => {
+                    trace!(target: "aleph-network", "Manager Loop message from network case");
+                    match maybe_message {
```

Comment:
Isn't the linting broken here? Also above? The indentation seems wrong.

Reply:
You are right, done